display_applications/ucsc/trackhub.xml

```diff
@@ -0,0 +1,6 @@
+<display id="ucsc_trackhub" version="1.0.0" name="display at Track Hub UCSC">
+    <link id="main" name="main">
+        <url>https://genome.ucsc.edu/cgi-bin/hgHubConnect?hubUrl=${qp($hub_file.url + '/myHub/hub.txt')}&amp;hgHub_do_firstDb=on&amp;hgHub_do_redirect=on&amp;hgHubConnect.remakeTrackHub=on</url>
+        <param type="data" name="hub_file" url="galaxy_${DATASET_HASH}" allow_extra_files_access="True" strip_https="True"/>
```

Comment:
It swaps https to http (when Galaxy is under https) for the url that Galaxy gives to the external display application. At one point the UCSC genome browser was not able to fetch https URLs; I do not think this is the case any longer.